docs/source/usage.rst

```diff
@@ -214,6 +226,15 @@ Example:
   1 snp_c 65429279
   1 snp_d 107421759
 
+Alternative Allele Frequency File
+=================================
+
+The alternative allele frequency file allows for user-defined population alternative allele frequency or frequencies where metafounders are used. The alternative allele frequency file contains the metafounder group denoted MF_x, where x is a numerical value, and the associated alternative allele frequency. If any alternative allele frequencies are missing from the file, the default of 0.5 will be used for these metafounders.
```

Comment:
@RosCraddock change to 

```
The alternative allele frequency file allows for user-defined population alternative allele frequencies. This file contains the metafounder group denoted MF_x, where x is by default "1" but see ``-main_metafounder``, followed by alternative allele frequencies for all the markers. In case of multiple metafounders, provide multiple rows in the file. The default starting alternative allele frequencies are 0.5. If you don't have information for some markers, provide 0.5 for these in the file.
```